changelogs/5147-terraform-init-no-color.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - terraform - run ``terraform init`` with ``-no-color`` not to mess up the stdout of the task.
```

```suggestion
  - terraform - run ``terraform init`` with ``-no-color`` not to mess up the stdout of the task (https://github.com/ansible-collections/community.general/pull/5147).
```
Please note that this file must be in the subdirectory `fragments/` of `chanelogs/`.